packages/jaeger-ui/src/components/App/TopNav.test.js

```diff
@@ -111,32 +115,6 @@ describe('<TopNav>', () => {
       expect(item.length).toBe(1);
       expect(item.find(`[target="_blank"]`).length).toBe(1);
     });
-
-    describe('<CustomNavDropdown>', () => {
-      let subMenu;
-
-      beforeEach(() => {
-        wrapper = shallow(<TopNav.CustomNavDropdown {...configMenuGroup} />);
-        subMenu = shallow(wrapper.find('Dropdown').props().overlay);
-      });
-
-      it('renders sub-menu text', () => {
-        dropdownItems.slice(0, 0).forEach(itemConfig => {
-          const item = subMenu.find(`[text="${itemConfig.label}"]`);
-          expect(item.length).toBe(1);
-          expect(item.prop('disabled')).toBe(true);
-        });
-      });
-
-      it('renders sub-menu links', () => {
-        dropdownItems.slice(1, 2).forEach(itemConfig => {
-          const item = subMenu.dive().find(`[href="${itemConfig.url}"]`);
-          expect(item.length).toBe(1);
-          expect(item.prop('target')).toBe(itemConfig.anchorTarget || '_blank');
-          expect(item.text()).toBe(itemConfig.label);
-        });
-      });
-    });
```

Comment:
So basically we're only relying on the typing now (`MenuProps`)?

Reply:
Yes. But not specifically `MenuProps`. DropDown items use `ConfigMenuItem` type from `src/types/config.tsx`